src/neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -191,6 +212,12 @@ public void DeserializeUnsigned(BinaryReader reader)
             }
             Script = reader.ReadVarBytes(ushort.MaxValue);
             if (Script.Length == 0) throw new FormatException();
+            if (IsOracleResponse)
+            {
+                if (Cosigners.Count > 0) throw new FormatException();
```

Comment:
Why not allow `cosigners`?

Reply:
Because the oracle response script is strictly limited in the next line. So no cosigners are needed. The cosigners of the request transaction will be copied to the response transaction to execute the callback.